doc/man3/SSL_CTX_set_cert_store.pod

```diff
@@ -16,7 +16,9 @@ SSL_CTX_set_cert_store, SSL_CTX_set1_cert_store, SSL_CTX_get_cert_store - manipu
 
 SSL_CTX_set_cert_store() sets/replaces the certificate verification storage
 of B<ctx> to/with B<store>. If another X509_STORE object is currently
-set in B<ctx>, it will be X509_STORE_free()ed.
+set in B<ctx>, it will be X509_STORE_free()ed. SSL_CTX_set_cert_store() will
+take ownership of the B<store>, i.e., the call X509_STORE_free(store) is no
```

Comment:
`C<X509_STORE_free(store)>`

Reply:
Applied

Reply:
> C<X509_STORE_free(store)>

@t8m Can you please confirm that it's rendered as expected? I see that now the `X509_STORE_free(store)` definitely stands out from other similar objects like `X509_STORE_free()` or `SSL_CTX_set_cert_store()`:

![openssl_screen](https://github.com/openssl/openssl/assets/4346993/ce797c02-1130-4941-8209-d7dfc863ac31)

- https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_cert_store.html

Reply:
IMO this is correct. `X509_STORE_free(store)` is a code snippet where those functions are just names of functions.